constellations/orion/src/domains/user_afh.go

```diff
@@ -0,0 +1,28 @@
+package domains
+
+import "errors"
+
+var TABLE_USERAFH = "user_afh"
+
+type UserAfh struct {
+	Id     uint `json:"id"`
+	UserId uint `json:"user_id"`
```

Comment:
I think json here should be `userId` and the db should be `user_id`. Look at other domains for examples.